### DISCUSSION TITLE
期待付费版 这是一个非常棒的主题

### DISCUSSION THREAD

**Discussion by shaoyangshujian** [Tokinx/Wing#53] [讨论]
另外我的写作日历永远都是Days 1 也不显示更新小亮点
非常苦恼这个问题
https://chngp.com

另外一个建议就是首页的标签栏“全部 笔记 文章”这三个按钮可以设置其中一个不显示，比如我从来不用笔记功能就不希望显示笔记这个空白页。

**Comment by Tokinx** [comment]
days是根据你发布的第一条“笔记”时间开始计算的，你不用笔记这个功能所以一直是days1，这个我看一下如何处理比较好。（第一篇文章或第一篇笔记到当前时间取最长的Days）
如果关闭“笔记”标签栏，那么这个导航获取就没意义了，我加一个隐藏标签栏的功能？（仅在未登录时隐藏，登陆后显示）

**Comment by Tokinx** [comment]
功能实现方案可以在这个帖子下面继续讨论，我会抽时间完善这些地方。
精力有限，所以主题没有收费计划，如果喜欢这款主题可以打赏支持一下。

**Reply by shaoyangshujian** [reply]
有没有群啊。在哪里有支付赞赏的地方？

**Reply by shaoyangshujian** [reply]
只是想表达微薄心意。

**Reply by shaoyangshujian** [reply]
找到支付宝二维码了。感谢作者的劳动与努力。
